src/main/java/frc/robot/commands/ForgetArmHomeCommand.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands;
```

Comment:
Suggest naming this ArmElbowForgetHomesCommand since involves both subsystems

Reply:
good idea. done.